test/utils/StakingNodeTestBase.sol

```diff
@@ -197,6 +198,9 @@ contract StakingNodeTestBase is ScenarioBaseTest, ProofParsingV1 {
 
         vm.roll(block.number + delegationManager.minWithdrawalDelayBlocks() + 1);
 
+        vm.expectRevert(bytes4(keccak256("NotStakingNodesOperator()")));
```

Comment:
don't add specific testing of error cases within other tests.

Tests these in separate tests whose name describes what's being tested

Reply:
fixed in https://github.com/yieldnest/yieldnest-protocol/pull/124/commits/e40cb6f14cb1feb306eafa7a9ba859b03944625e.